packages/app/src/sections/dashboard/HistoryTabs.tsx

```diff
@@ -0,0 +1,87 @@
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import TabButton from 'components/Button/TabButton'
+import { TabPanel } from 'components/Tab'
+import Trades from 'sections/futures/Trades'
+import TraderHistory from 'sections/leaderboard/TraderHistory'
+import { useAppSelector } from 'state/hooks'
+import { selectWallet } from 'state/wallet/selectors'
+import media from 'styles/media'
+
+export enum HistoryTab {
+	Positions = 'positions',
+	Trades = 'trades',
+}
+
+type HistoryTabsProp = {
+	currentTab: HistoryTab
+	onChangeTab(tab: HistoryTab): () => void
+}
+
+const HistoryTabs: React.FC<HistoryTabsProp> = ({ currentTab, onChangeTab }) => {
+	const { t } = useTranslation()
+	const walletAddress = useAppSelector(selectWallet)
+
+	return (
+		<StakingTabsContainer>
+			<StakingTabsHeader>
+				<TabButtons>
+					<TabButton
+						variant="noOutline"
+						title={t('dashboard.history.tabs.positions')}
+						onClick={onChangeTab(HistoryTab.Positions)}
+						active={currentTab === HistoryTab.Positions}
+					/>
+					<TabButton
+						variant="noOutline"
+						title={t('dashboard.history.tabs.trades')}
+						onClick={onChangeTab(HistoryTab.Trades)}
+						active={currentTab === HistoryTab.Trades}
+					/>
+				</TabButtons>
+			</StakingTabsHeader>
+			<div>
+				<TabPanel name={HistoryTab.Positions} activeTab={currentTab}>
+					<TraderHistory trader={walletAddress!} resetSelection={() => {}} compact={true} />
```

Comment:
Not sure we should add the `!` as there can be no wallet address

Reply:
True. Using the nullish coalescing operator might be more appropriate.